airbyte-integrations/connectors/source-googleanalytics-singer/build.gradle

```diff
@@ -0,0 +1,20 @@
+project.ext.pyModule = 'googleanalytics_singer_source'
```

Comment:
nit: can we follow the same naming pattern as the dir: `source-googleanalytics-singer`?